include/cudaq/Optimizer/Transforms/Passes.h

```diff
@@ -46,9 +46,10 @@ std::unique_ptr<mlir::Pass> createRaiseToAffinePass();
 std::unique_ptr<mlir::Pass> createUnwindLoweringPass();
 
 std::unique_ptr<mlir::Pass>
-createPySynthCallableBlockArgs(const std::vector<std::string> &);
+createPySynthCallableBlockArgs(const std::vector<std::string> &,
```

Comment:
Should be using `SmallVector`, `SmallVectorImpl`, or `ArrayRef` here, no? Maybe `StringRef`?

Reply:
fixed